plugin-api/src/main/java/org/sonar/api/batch/fs/InputFile.java

```diff
@@ -194,6 +194,11 @@ enum Status {
    */
   Charset charset();
 
+  /**
+   * Hexadecimal MD5 digest of the file's content.
+   */
```

Comment:
Missing @since

Reply:
Should we clarify if this is the hash of the original file content or the hash of the same content as returned by `content()` or `inputStream()` (= without the BOM)

Reply:
> Should we clarify if this is the hash of the original file content or the hash of the same content as returned by `content()` or `inputStream()` (= without the BOM)

Good point. We discussed it during the last guild weekly, it will be clarified by @dbmeneses

Reply:
Sorry for the delay. 
Looking at the code, I believe the hash is the hash of the contents of the file as returned by `content()` and `inputStream()`, without the BOM if there's one.